### PR TITLE
Added Dockerfile

### PR DESCRIPTION
**Pull request by t04glovern**
## Description
 - Includes docker, docker-compose and .dockerignore files for a clean experience
 - README outlining the use

 ## _Issue_ Related
#67 

 ## Check list:
- [x] My code follows the code style of this project.
- [x] My _commits_ follow the style pattern of this project.
- [x] My change requires a change in documentation.
- [x] We updated the documentation accordingly.
- [x] I have read the Contribution document (**CONTRIBUTING**).
- [ ] I've added tests to cover my changes.
- [ ] All new and existing tests have passed.

### CODE REVIEW

**Review by bossbossk20 — CHANGES_REQUESTED** [review]
Can you change to image of node to alpine version

**Comment by bossbossk20** [comment]
Everything looks ok but can you change to node alpine version

**Comment by t04glovern** [comment]
> Everything looks ok but can you change to node alpine version

Should be all converted, tested using alpine and works fine on my end.

File: README.md [hunk 1]
```diff
@@ -2,43 +2,63 @@
...
+
+```bash
+docker-compose stop
+```
+
 ## How to Contribute
+
 A. Open an issue
 
-   ONLY REQUEST A FIX FOR AN ISSUE THAT DOESNT EXIST,
```
Comment (t04glovern) [inline README.md:33]: Sorry about these, the precommit in the package.json auto formatted.

**Review by bossbossk20 — APPROVED** [review]
Thank you for your PR

**Pull request merged**